README.md

```diff
@@ -21,9 +21,9 @@ The Submitty project is hosted on [GitHub](https://github.com/Submitty).
 # Key Features
 
 * Secure testing of many languages: Python, C/C++, Java, Scheme, Prolog, SPIM, and anything available on GNU / Linux!
-* Customizable automated grading with immediate feedback to students
+* Customizable automated grading with immediate feedback to students.
 * Advanced grading tools: static analysis, JUnit, code coverage, memory debuggers, etc.
-* Student upload by drag-and-drop, zip upload, or version control
+* Student upload by drag-and-drop, zip upload, or version control.
 * Correct mistakes through multiple submissions, flexible ``late day’’ policy, hidden tests
```

Comment:
These quotes do not match. Should be double quotes.

Reply:
Fixed in master, separate from this PR.